### ISSUE TITLE
There should be a way to leave a chat

### ISSUE THREAD

**Issue by bsteinbeigle** [Methodician/guytime#85]

**Comment by bsteinbeigle** [comment]
Yes, I would say lets hold off on this for now. I would rather work on issue 75 for now unless working on the above will help support troubleshooting of issue 75.

**Comment by kaxline** [comment]
@bsteinbeigle Yeah, I think this is needed to better see what's going on in #75. Going to move it to In Progress.

**Comment by bsteinbeigle** [comment]
@kaxline Here is sketch I did for deleting messages. Let me know if you have any questions. 
![unnamed (3)](https://user-images.githubusercontent.com/89544670/136600637-4fa34b07-7c55-44ad-b3f7-2ac1623e75d8.jpg)
